integration/client/import_test.go

```diff
@@ -79,7 +79,7 @@ func testExportImport(t *testing.T, imageName string) {
 	}
 	defer client.Close()
 
-	_, err = client.Fetch(ctx, imageName)
+	_, err = client.Fetch(ctx, imageName, WithPlatformMatcher(platforms.Default()))
```

Comment:
Maybe we can push that image into ghcr.io in our org? just like https://github.com/containerd/containerd/pkgs/container/busybox

Reply:
Yeah I think that makes sense to just remove this mcr dependency. I'll take a look at how to push the image to our ghcr.

(Meanwhile do you think we can merge this fix to unblock faster? :))